src/qseed.cpp

```diff
@@ -8,74 +8,119 @@
 #include <sstream>
 #include <thread>
 
-const unsigned long DEFAULT_QSEED_SIZE = 2;
-const unsigned long DEFAULT_QSEED_PERIOD = 10;
+const uint32_t MAX_QSEED_SIZE = 512*1024;          // 512 kib
+const uint32_t MAX_QSEED_PERIOD = 60*60*24*365;    // 1 year
+const uint32_t DEFAULT_QSEED_SIZE = 64000;
```

Comment:
I thought I read that you typically provide like 128bits when you call C_SeedRandom. Where did you get 64k from?

Reply:
Eric from Thales responded in an email that the maximum amount that can be injected into Thales HSMs is 64kb. I'm starting with this value. We can modify it later once we test it and get customer requirements.

Where did you get 128 bits from?

Reply:
Thanks Kenny for sharing those documents. 

I read the Thales Luna document about DRBGs. They use AES-256 CTR DRBG for Luna HSM 7. After reading the NIST 800-90A document, this means that the seed length is 384 bits. The seed is padded with zeros if the additional input is less than 384 bits. The additional input is passed into a derivation function if the length is larger than 384 bits. Ultimately, the reseed length will be 384 bits.

Therefore, I'm setting the default QSEED_SIZE to 384 bits (48 bytes) for now.